### PR TITLE
Update content.md

### PR DESCRIPTION
**Pull request by marqdevx**
- We need to add images
-

### CODE REVIEW

File: content/tutorials/portenta-h7/por-ard-lvgl/content.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+-   Understanding the structure to build LVGL interfaces.
+-   Building a simple UI with a text label and a button. 
 -   Configuring the setup to display the User-Interface. 
 
 ## Required Hardware and Software
 -   Portenta H7 board (<https://store.arduino.cc/portenta-h7>)
 -   USB C cable (either USB A to USB C or USB C to USB C)
 -   Arduino IDE 1.8.10+  or Arduino Pro IDE 0.0.4+ 
--   USB 
+-   USB-C hub with HDMI
```
Comment (Lenardgeorge) [inline content/tutorials/portenta-h7/por-ard-lvgl/content.md:13]: Can you provide the links to the Hub we are using ?

File: content/tutorials/portenta-h7/por-ard-lvgl/content.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Graphical User interfaces are necessary for visualising information and interact
 
 # Building a simple GUI 
 
-we are going to work on the **Portenta_lvgl** example , The Portenta H7 comes with an inbuilt,  In this tutorial we are going to upload the lvgl example to the portenta board. Once the sketch is uploaded the board becomes a usb host where it then is connected to the usb hub which then is connected to an external monitor and powered externally. when connected the monitor the buttons and text fields will be connected. 
+We are going to build a sketch from scratch,  In this tutorial we are going to upload our sketch to the portenta board. Once the sketch is uploaded the board becomes a usb host where it then is connected to the usb hub which then is connected to an external monitor and powered externally. when connected the monitor the buttons and text fields will be connected.
```
Comment (Lenardgeorge) [inline content/tutorials/portenta-h7/por-ard-lvgl/content.md]: This tutorial will guide you to build a basic user interface using the LVGL and the USBhost Libraries that you will have download using the Arduino Library Manager. The setup for this tutorial requires you to first to upload the finished sketch file to the Portenta board where it converts the board into a usb host Device. You will then connect the board to a USB-hub and connect an external monitor. Once the HUB is powered externally, a graphical user interface with a button and a text-field will be displayed on the screen.

Comment (Lenardgeorge) [reply content/tutorials/portenta-h7/por-ard-lvgl/content.md]: you can replace the text with what I wrote above

File: content/tutorials/portenta-h7/por-ard-lvgl/content.md [hunk 3]
```diff
@@ -37,15 +37,12 @@ we are going to work on the **Portenta_lvgl** example , The Portenta H7 comes wi
...
+   *** Make sure you have selected the right board inside Tools > Boards ***
    
-   ![por_ard_lvgl_select_example](assets/por_ard_lvgl_select_example.svg)
    
    
    
-3. The Hello World label widget 
-
-   Lets start by including the libraries that we are going to use.
+3. Including the libraries that we are going to use.
```
Comment (Lenardgeorge) [inline content/tutorials/portenta-h7/por-ard-lvgl/content.md]: Shall we combine step 3 with 4 and have something like this 

3. Adding Buttons and Labels
Lets start by including the `Portenta_LittleVGL.h` and `USBHost.h` libraries.  
```cpp
 #include "Portenta_LittleVGL.h"
 #include "USBHost.h"
```
We will then create a button widget using the `lv_btn` object. This object also allows you to have an inner label whose text can be easily configured. The following lines of code initialises the button variable and its label, `myCustomLabel` which you can add it at the beginning of the sketch.

File: content/tutorials/portenta-h7/por-ard-lvgl/content.md [hunk 4]
```diff
@@ -54,18 +51,15 @@ we are going to work on the **Portenta_lvgl** example , The Portenta H7 comes wi
...
    static lv_obj_t *lv_btn;
    static lv_obj_t *myCustomLabel;
    ```
-
-   ![por_ard_lvgl_create_variables](assets/por_ard_lvgl_create_variables.png)
-
    
 
-   Then at the end of the `setup()` you can configure the type, style and the position of the button. 
+   Then at the end of the `setup()` you can configure the type, style and the position of the button and label.
```
Comment (Lenardgeorge) [inline content/tutorials/portenta-h7/por-ard-lvgl/content.md]: If you are combining 3 and 4 then this step can be called as "Configuring the button widget"

File: content/tutorials/portenta-h7/por-ard-lvgl/content.md (hunk 3 above)
Comment (Lenardgeorge) [reply content/tutorials/portenta-h7/por-ard-lvgl/content.md]: You can then have a separate section for the `void setup()`part as step 4

File: content/tutorials/portenta-h7/por-ard-lvgl/content.md [hunk 5]
```diff
@@ -78,10 +72,19 @@ we are going to work on the **Portenta_lvgl** example , The Portenta H7 comes wi
...
+      // put your main code here, to run repeatedly:
+      lv_task_handler();
+   }
+   ```
 
-   Compile and Upload the sketch to your board. Connect the board to the USB hub with the Monitor attached to the HDMI port.  Once you power up the HUB, you will see a button with a label **CustomLabel**.
+   Compile and Upload the sketch to your board. Connect the board to the USB hub with the Monitor attached to the HDMI port.  Once you power up the HUB, you will see a button with a label saying **Custom Button**.
 
    
 4. Connect an external monitor
```
Comment (Lenardgeorge) [inline content/tutorials/portenta-h7/por-ard-lvgl/content.md:78]: This should be step 5

File: content/tutorials/portenta-h7/por-ard-lvgl/content.md [hunk 6]
```diff
@@ -78,10 +72,19 @@ we are going to work on the **Portenta_lvgl** example , The Portenta H7 comes wi
...
+   
+   ```cpp
+   void loop() {
+      // put your main code here, to run repeatedly:
+      lv_task_handler();
+   }
+   ```
 
-   Compile and Upload the sketch to your board. Connect the board to the USB hub with the Monitor attached to the HDMI port.  Once you power up the HUB, you will see a button with a label **CustomLabel**.
+   Compile and Upload the sketch to your board. Connect the board to the USB hub with the Monitor attached to the HDMI port.  Once you power up the HUB, you will see a button with a label saying **Custom Button**.
```
Comment (Lenardgeorge) [inline content/tutorials/portenta-h7/por-ard-lvgl/content.md]: I would move this paragraph to step 5 which will have the following substeps / flow of steps in the following order  
1. Upload sketch to board 
2. Disconnect board 
3. Connect the board to hub
4. Connect the monitor to hub
5. Power up the hub
6. See the display

**Review by Lenardgeorge — COMMENTED** [review]
Went through the steps and added missing information and reorganised the steps

File: content/tutorials/portenta-h7/por-ard-lvgl/content.md (hunk 5 above)
Comment (sbhklr) [reply content/tutorials/portenta-h7/por-ard-lvgl/content.md:78]:
```suggestion
5. Connect an External Monitor
```

**Pull request merged**